commands/types/fakedoor/message.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var messageFmt = "The support for %s hasn't been implemented yet.\nIf you would like us to implement it, please drop by at:\n%v and add thumbs up.\nWe're happy to implement it quickly based on demand!"
+var IssueURLFmt = "https://github.com/hazelcast/hazelcast-commandline-client/issues/%d"
+
+type FakeDoor struct {
+	Name     string
+	IssueNum int
+}
+
+func MakeFakeCommand(fd FakeDoor) *cobra.Command {
+	return &cobra.Command{
+		Use: fd.Name,
```

Comment:
Can you also add field named "Short"? Currently output of `hzc help ` look incomplete.

Reply:
Thanks, fixed [6f49af6](https://github.com/hazelcast/hazelcast-commandline-client/pull/45/commits/6f49af6098d1a9729952ba84f745aa75ae657b09).